orange/src/main/java/com/hartwig/hmftools/orange/algo/pave/PaveAlgo.java

```diff
@@ -21,10 +21,12 @@ public class PaveAlgo
 
     @NotNull
     private final EnsemblDataCache ensemblDataCache;
+    private boolean expectGenesExist;
 
     public PaveAlgo(@NotNull final EnsemblDataCache ensemblDataCache)
     {
         this.ensemblDataCache = ensemblDataCache;
+        expectGenesExist = true;
```

Comment:
Could this be a parameter in an overloaded constructor and made final?

Reply:
Done